handlers/analytics/handler_test.go

```diff
@@ -0,0 +1,72 @@
+package analytics
```

Comment:
added test to cover URL param missing. If any of the other params are missing I don't think should do anything other than redirect the user to the link they selected. ATM the analytics service doesn't do anything so there is nothing to test.